{"name": "Get Personagens", "status": "failed", "statusDetails": {"message": "ConnectionError: HTTPSConnectionPool(host='gateway.marvel.com', port=443): Max retries exceeded with url: /v1/public/characters?ts=1631150721003&apikey=e9e22f92431df9f73920f281caabb68c&hash=f0b3298fa4957047893445f8d19881b8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb3dc937280>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))"}, "steps": [{"name": "RequestsLibrary.GET", "status": "failed", "statusDetails": {"message": "ConnectionError: HTTPSConnectionPool(host='gateway.marvel.com', port=443): Max retries exceeded with url: /v1/public/characters?ts=1631150721003&apikey=e9e22f92431df9f73920f281caabb68c&hash=f0b3298fa4957047893445f8d19881b8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb3dc937280>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))"}, "attachments": [{"name": "Keyword Log", "source": "9362785a-8724-4d56-afb7-a3ec5060427a-attachment.html", "type": "text/html"}], "parameters": [{"name": "arg1", "value": "https://gateway.marvel.com/v1/public/characters?"}, {"name": "arg2", "value": "${headers}"}], "start": 1631202988622, "stop": 1631203108661}], "start": 1631202988621, "stop": 1631203108662, "uuid": "4a51e8db-57bd-4603-96e9-43d49f4ba618", "historyId": "a620551ea1b8dedd4c64f609992042e9", "fullName": "Search Characters.Get Personagens", "labels": [{"name": "suite", "value": "Search Characters"}, {"name": "framework", "value": "robotframework"}, {"name": "language", "value": "cpython3"}, {"name": "host", "value": "C0167.local"}, {"name": "thread", "value": "default-b4cb847f-d124-40b0-877d-89ea304d866e"}]}